core/src/test/java/com/sequenceiq/cloudbreak/init/blueprint/SafetyValveFormatValidatorTest.java

```diff
@@ -0,0 +1,88 @@
+package com.sequenceiq.cloudbreak.init.blueprint;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Stream;
+
+import org.apache.commons.io.FileUtils;
+import org.apache.commons.io.filefilter.DirectoryFileFilter;
+import org.apache.commons.io.filefilter.RegexFileFilter;
+import org.apache.commons.lang3.StringUtils;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.sequenceiq.cloudbreak.cmtemplate.CmTemplateProcessor;
+import com.sequenceiq.cloudbreak.cmtemplate.utils.BlueprintUtils;
+import com.sequenceiq.cloudbreak.cmtemplate.validation.SafetyValveValidator;
+import com.sequenceiq.cloudbreak.json.JsonHelper;
+
+@ExtendWith(MockitoExtension.class)
+public class SafetyValveFormatValidatorTest {
+
+    private BlueprintUtils blueprintUtils = new BlueprintUtils();
+
+    private JsonHelper jsonHelper = new JsonHelper();
+
+    private SafetyValveValidator safetyValveValidator = new SafetyValveValidator();
```

Comment:
These are stateless so i defined here to avoid creating objects. I have moved them to setup now. since it's possible in future they have some state and that can cause issue.

Reply:
Yes, that is exactly why I suggested so. :smile: They can be stateless now, but that may change any time in the future. The UT shall not assume anything / trust any classes it depends on, except for the victim / `underTest` instance of course.